yarn.lock

```diff
@@ -1597,6 +1602,36 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.42.0.tgz#6b7ca87ed30a92634e72cbe8fc1280fd21db50b8"
   integrity sha512-hBPCikSDJ3Rgw+mAbTgnNB83dn3SeSKPo4XNli22tbV08fN8P5mEaEg8Uw0cbVMN3D138ykNeAaK/lZjO9KIqA==
 
+"@fluentui-react-native/framework@0.8.33":
```

Comment:
Having FURN packages in yarn.lock usually means you're using outdated versions in a package.json file. Please update to align your version with the repo.